uid2_client/request_response_util.py

```diff
@@ -0,0 +1,41 @@
+import base64
+import os
+from urllib import request
+
+import pkg_resources
+
+from uid2_client.encryption import _encrypt_gcm, _decrypt_gcm
+
+
+def _make_url(base_url, path):
+    return base_url + path
+
+
+def auth_headers(auth_key):
+    return {'Authorization': 'Bearer ' + auth_key,
+            "X-UID2-Client-Version": "uid2-client-python-" + pkg_resources.get_distribution("uid2_client").version}
```

Comment:
i can't remember who added this but when i run the python source code raw without packaging and using pypi to install uid2_client package this caused issue in test_client.py:

```
Error
Traceback (most recent call last):
  File "/usr/lib/python3.8/unittest/mock.py", line 1348, in patched
    return func(*newargs, **newkeywargs)
  File "/mnt/c/dev/uid2-client-python/tests/test_client.py", line 121, in test_uid2_client_produces_uid2_token
    client.refresh_keys()
  File "/mnt/c/dev/uid2-client-python/uid2_client/client.py", line 83, in refresh_keys
    resp = post(self._base_url, '/v2/key/sharing', headers=auth_headers(self._auth_key), data=req)
  File "/mnt/c/dev/uid2-client-python/uid2_client/request_response_util.py", line 16, in auth_headers
    "X-UID2-Client-Version": "uid2-client-python-" + pkg_resources.get_distribution("uid2_client").version}
  File "/root/.virtualenvs/uid2-client-python/lib/python3.8/site-packages/pkg_resources/__init__.py", line 478, in get_distribution
    dist = get_provider(dist)
  File "/root/.virtualenvs/uid2-client-python/lib/python3.8/site-packages/pkg_resources/__init__.py", line 354, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/root/.virtualenvs/uid2-client-python/lib/python3.8/site-packages/pkg_resources/__init__.py", line 909, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/root/.virtualenvs/uid2-client-python/lib/python3.8/site-packages/pkg_resources/__init__.py", line 795, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'uid2_client' distribution was not found and is required by the application
```

How did you get around this? Should we add some try/except to avoid crashing and use a default value like "non-packaged-mode"?

@cody-constine-ttd thought?